### PR TITLE
chore: allow reassignment of properties on arguments

### PR DESCRIPTION
**Pull request by haxxmaxx**

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -19,7 +19,8 @@
     "react/forbid-prop-types": 0,
     "react/jsx-props-no-spreading": 0,
     "import/no-extraneous-dependencies": [2, { "devDependencies": true }],
-    "prefer-destructuring": ["error", { "object": true, "array": false }]
+    "prefer-destructuring": ["error", { "object": true, "array": false }],
+    "no-param-reassign": [2, { "props": false }]
```
Comment (LiKang6688) [inline .eslintrc.json:23]: what does 2 mean here?

Comment (LiKang6688) [reply .eslintrc.json:23]: found it https://eslint.org/docs/2.0.0/rules/no-param-reassign, but it does not explain that...

**Review by LiKang6688 — APPROVED** [review]

File: .eslintrc.json (hunk 1 above)
Comment (haxxmaxx) [reply .eslintrc.json:23]: Don't know honestly, I copy pasted it and made sure it works. Still gives error if I would reassign the entire arg (`someArg = newValue`).

Comment (LiKang6688) [reply .eslintrc.json:23]: > Don't know honestly, I copy pasted it and made sure it works. Still gives error if I would reassign the entire arg (`someArg = newValue`).

Never mind. Cool.

**Pull request merged**